src/components/Homepage/Homepage.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react';
+import * as styles from '../../styles/HomePage.module.css';
+import { Header } from '../Header/Header.js';
+import { Footer } from '../Footer/Footer.js';
+import { Nav } from './Nav/Nav.js';
+
+function HomePage(props) {
+  return (
+    <>
+      <Header />
```

Comment:
would you want to reuse the header and footer for other pages without adding the component to each page? Where will you put it to be able to do that?

Reply:
Oh, right, of course. If we're gonna use the same header and footer, I should put the components in App.jsx